ocaml/idl/datamodel_vm.ml

```diff
@@ -214,17 +214,13 @@ let power_behaviour =
   let snapshot_with_quiesce = call
       ~name:"snapshot_with_quiesce"
       ~in_product_since: rel_orlando
```

Comment:
Please replace this `~in_product_since` with a `~lifecycle` arg that has `Published` and `Removed` entries.

Reply:
Adding `Removed` will trigger a standard `MESSAGE_REMOVED` error to be raised, so you don't need to define a new one.